packages/pds/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@atproto/pds",
   "version": "0.0.2",
-  "main": "src/index.ts",
+  "main": "dist/index.js",
```

I expect some script that runs on install must be touching these.
```suggestion
  "main": "src/index.ts",
```